test/REQUIRE

```diff
@@ -0,0 +1,2 @@
+JuMP
```

Comment:
Oh nice, I didn't know you could have a separate REQUIRE for tests in Julia packages.